verifier/src/x509/chain.rs

```diff
@@ -260,3 +270,67 @@ mod test {
         );
     }
 }
+
+#[cfg(test)]
+mod pkits {
```

Comment:
My thought was to separate the pkits tests from the others.

And as I write this perhaps the pkits tests belong as integration tests...???  I'm up for suggestions there.

Reply:
Moved to integration tests in #75